.lintrunner.toml

```diff
@@ -168,3 +168,36 @@ command = [
     '@{{PATHSFILE}}'
 ]
 is_formatter = true
+
+[[linter]]
+code = 'CLANGFORMAT'
+include_patterns = [
+    '**/*.h',
+    '**/*.cc',
```

Comment:
could we also include "*.m" and "*.mm" (Objective-C and C++) too?

Reply:
ok!

Reply:
Done